.github/workflows/flake8.yml

```diff
@@ -23,4 +23,4 @@ jobs:
       - name: install flake8
         run: pip install -r requirements-dev.txt
       - name: run flake8
-        run: flake8 src/ setup.py
+        run: flake8 --ignore=E501,W503 src/ setup.py
```

Comment:
Consider adding the following lines to setup.cfg: https://github.com/plone/meta/blob/06c445cd79552555c248e2e97af49f99f280236d/config/default/setup.cfg.j2#L18-L28
In this way you will not need to specify the parameters for flake8.